invenio_communities/views/api.py

```diff
@@ -25,9 +25,15 @@ def init(state):
 
     # services
     rr_ext.registry.register(ext.service)
+    rr_ext.registry.register(ext.service.members, service_id="community-members")
```

Comment:
```suggestion
    rr_ext.registry.register(ext.service.members, service_id="members")
```

to be consistent with indexer naming?

Reply:
will amend the commit with the change on monday

Reply:
updated, and explicitly set the `service_id` for the communities service too, for consistency within this function